app/controllers/healthcheck_controller.rb

```diff
@@ -27,6 +28,14 @@ def check
 
 private
 
+  def status
+    if database_connected? && database_populated?
+      :ok
+    else
+      :internal_server_error
```

Comment:
since we will be returning this, should Sentry ignore this endpoint as well?

Reply:
I'm happy for it to be ignored if it becomes a problem but I think for now just let everything through.